scripts/DDL.sql

```diff
@@ -0,0 +1,48 @@
+
+CREATE TABLE users(
+	upn text PRIMARY KEY,
+	display_name text
+);
+
+CREATE TABLE tags(
+	id serial PRIMARY KEY,
+	name text
+);
+
+CREATE TABLE roles(
+	id serial PRIMARY KEY,
+	name text
+);
+
+CREATE TABLE standards(
+	id serial PRIMARY KEY,
+	name text
+);
+
+CREATE TABLE advertisements(
+	id serial PRIMARY KEY,
+	role_id int REFERENCES roles(id) ON DELETE CASCADE,
+	unit text,
+	branch text,
+	department text,
+	description text,
+	standard_id int REFERENCES standards(id) ON DELETE CASCADE,
+	entry_date Date,
+	seniority int,
+	is_damach boolean,
+	advertiser_upn text REFERENCES users(upn) ON DELETE CASCADE,
+	contact text,
+	base_location text
+);
+
+CREATE TABLE tags_of_adds(
```

Comment:
what is the meaning of tag?

Reply:
https://user-images.githubusercontent.com/62653606/77565962-a8f1ee80-6ecd-11ea-9bda-45c4ee526643.png

Look at this 😄

Reply:
Ok I got it for every addvisiter you can add few tags, Stav told me it will be used for thh search